### ISSUE TITLE
Eccentricity deformation

### ISSUE THREAD

**Issue by ghost** [areejalghamdi/GraphicsT1#1]
Do we want the relative positions of the eyes, nose, mouth, etc. to remain the same when we change the head's eccentricity? Or should they maintain their absolute position?

That is, if we make the head wider, should the eyes also spread apart so that they have the same relative spacing? Or should the eyes remain in the same position, which could cause widely spaced eyes to appear "narrow" when the head is widened.

**Comment by Kentrosaurus** [comment]
Yes,  changing any single degree of freedom should not affect any other.  So the head eccentricity should just affect the overall height-width ratio, but the eyes, nose and mouth sizes and spacing, etc. should remain unchanged.  No coupling or cross-talk between or among the various parameters.

Making head wider leaves eye spacing unchanged.

Kent

**Comment by areejalghamdi** [comment]
The weight painting tool is what I used to ease the effect of the deformation on the mouth, eye spacing, etc.
